packages/donate-button-next/src/components/widget/api/get-nonprofit-info.ts

```diff
@@ -1,33 +1,47 @@
 import {DonationRecipient} from 'src/components/widget/types/donation-recipient';
 import {NonprofitInfo} from 'src/components/widget/types/nonprofit-info';
-import {PaymentMethod} from 'src/components/widget/types/payment-method';
 
-const BASE_EVERY_URL = 'https://partners.every.org/v0.2/nonprofit/';
+const BASE_EVERY_URL = 'https://partners.every.org/v0.2/nonprofit';
 const BASE_CLOUDINARY_URL =
 	'https://res.cloudinary.com/everydotorg/image/upload/';
 
 const getCloudinaryUrl = (filename: string) => {
 	return `${BASE_CLOUDINARY_URL}${filename}`;
 };
 
-interface EveryResponse {
-	message: string;
-	data: {
-		nonprofit: EveryNonprofit;
-	};
-}
-
 const countriesMock: DonationRecipient[] = [];
 
-interface EveryNonprofit {
+type EveryFundraiser = {
+	id: string;
+	title: string;
+	description: string;
+	coverImageCloudinaryId: string;
+};
+
+type EveryNonprofit = {
 	name: string;
 	logoCloudinaryId: string;
 	coverImageCloudinaryId: string;
 	description: string;
 	eligibleDonationRecipientNonprofits?: DonationRecipient[];
 	locationAddress: string | null;
-}
+};
 
+type EveryResponse = {
+	message: string;
+	data: {
+		nonprofit: EveryNonprofit;
+	};
+};
+
+type FundraiserResponse = {
+	data: {
+		fundraiser: EveryFundraiser;
+		nonprofits: Array<EveryNonprofit>;
```

Comment:
```suggestion
		nonprofits: EveryNonprofit[];
```

The linter complains about that syntax.